### PR TITLE
[Chore] Client test coverage improvement exercise scores

### PR DESCRIPTION
**Pull request by ivohashamov**
### Checklist
- [x] Client: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html).
- [x] Client: I added multiple integration tests (Jest) related to the features (with a high test coverage)

### Motivation and Context

[Client test coverage improvment](https://confluence.ase.in.tum.de/display/ArTEMiS/Client+Test+Coverage+Improvements)
### Description

I created a test class for the exercise-scores.component

### Test Coverage

exercise-scores.component.ts: 27.93% -> 85.59%

### CODE REVIEW

File: .idea/runConfigurations/Artemis__Server_.xml [hunk 1]
```diff
@@ -5,9 +5,9 @@
     <option name="ACTIVE_PROFILES" value="dev,bamboo,bitbucket,jira,artemis,scheduling,local" />
     <option name="VM_PARAMETERS" value="--illegal-access=warn" />
     <option name="ALTERNATIVE_JRE_PATH" />
-    <option name="SHORTEN_COMMAND_LINE" value="NONE" />
+    <option name="SHORTEN_COMMAND_LINE" value="ARGS_FILE" />
```
Comment (krusche) [inline .idea/runConfigurations/Artemis__Server_.xml:8]: please revert this change!

File: src/main/resources/config/application-dev.yml [hunk 2]
```diff
@@ -45,7 +45,7 @@ spring:
     jpa:
         database-platform: org.hibernate.dialect.MySQL8Dialect
         database: MYSQL
-        show-sql: false
+        show-sql: true
```
Comment (krusche) [inline src/main/resources/config/application-dev.yml:48]: please revert this change!

**Review by krusche — CHANGES_REQUESTED** [review]
revert the unrelated changes and have a look at the GitHub warnings on unused variables

**Review by imstreetspirit — DISMISSED** [review]
tests looks good. there are complaints for unused variables, it could be removed

File: src/test/javascript/spec/component/exercises/shared/exercise-scores.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+                { provide: LocalStorageService, useClass: MockSyncStorage },
+                { provide: SessionStorageService, useClass: MockSyncStorage },
+                DifferencePipe,
+            ],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(ExerciseScoresComponent);
+                component = fixture.componentInstance;
+                exerciseService = TestBed.inject(ExerciseService);
```
Comment (FrankeLukas) [inline src/test/javascript/spec/component/exercises/shared/exercise-scores.component.spec.ts:133]: Unused variable.

File: src/test/javascript/spec/component/exercises/shared/exercise-scores.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+                accountService = TestBed.inject(AccountService);
```
Comment (FrankeLukas) [inline src/test/javascript/spec/component/exercises/shared/exercise-scores.component.spec.ts:134]: Unused variable.

File: src/test/javascript/spec/component/exercises/shared/exercise-scores.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+                resultService = TestBed.inject(ResultService);
+                profileService = TestBed.inject(ProfileService);
```
Comment (FrankeLukas) [inline src/test/javascript/spec/component/exercises/shared/exercise-scores.component.spec.ts:136]: Unused variable.

**Review by FrankeLukas — APPROVED** [review]
Lgtm otherwise. (Just remove the lines with unused variables.)

**Review by nicolasruscher — DISMISSED** [review]
Inspected the code, looks good! 👍

**Pull request merged**